requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 ## required dependencies
-pandas==2.1.0rc0
+pandas>=2.1,<2.2
```

Comment:
note that you can _not_ use `pandas=2.1` here as I said we can in conda `.yml` files